hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpcengine/net/AsyncSocket_RpcTest.java

```diff
@@ -265,6 +286,7 @@ private AsyncSocket newClient(SocketAddress serverAddress) {
                 .set(SO_SNDBUF, SOCKET_BUFFER_SIZE)
                 .set(SO_RCVBUF, SOCKET_BUFFER_SIZE)
                 .setReader(new ClientAsyncSocketReader())
+                .setSSLEngineFactory(new DefaultSSLEngineFactory())
```

Comment:
This should be done conditionally. So in the test, there should be a parameter 'tlsEnabled' which can be set by a subclass of this test. Otherwise, we lose the ability to test plain sockets.